README.md

```diff
@@ -229,8 +229,8 @@ const config = {
       {
         test: /\.html$/,
         use: [
-          { loader: 'html-loader', options: { minimize: true } }
-          'posthtml-loader'
+          { loader: 'html-loader', options: { minimize: true } },
+          { loader: 'posthtml-loader' }
```

Comment:
@nicholaslee119 It's correct, but normally you only need the `{Object}` from, if you also intend to set options, the `{String}`form is valid here. That said I'm ok with both 😛 .Thx for the missing comma fix 👍